pbs_light.go

```diff
@@ -862,6 +867,35 @@ func serve(cfg *config.Configuration) error {
 		return fmt.Errorf("Prebid Server could not load data cache: %v", err)
 	}
 
+	// Validate currency set in config
+	if cfg.AdServerCurrency != "" {
+		currency.MustParseISO(cfg.AdServerCurrency)
+	}
+
+	// Load latest currency conversion rates in blocking call
+	currencyLoadStopSignal := make(chan os.Signal, 1)
+	signal.Notify(currencyLoadStopSignal, syscall.SIGTERM, syscall.SIGINT)
+
+	go func() {
```

Comment:
I am not sure what the purpose of this go function is. It looks like it is set up to block just as if it was inline rather than a goroutine, but let serve continue if a SIGTERM or SIGINT comes in. But in that case, does it matter if `serve()` is allowed to continue if signalled, as presumably `serve()` should stop and the server should shut down if the signal comes in. But if the code in the goroutine is still executing, the whole process cannot shut down until it is done.

Reply:
this is intended to be a blocking call to fetch the latest conversion rates. should not allow serve to continue if fetching the rates fails.